.github/workflows/ci.yml

```diff
@@ -92,6 +101,10 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           args: --check runtime/
 
+      - if: "!cancelled()"
+        name: uncrustify
+        run: uncrustify -c ./src/uncrustify.cfg -q --replace --no-backup $(find ./src/nvim -name "*.[ch]")
```

Comment:
```suggestion
        run: uncrustify -c ./src/uncrustify.cfg -q --replace --no-backup $(git ls-files -- ':src/nvim/*.h' ':src/nvim/*.c')
```

Reply:
also, shouldn't this be using `--check` instead?

Reply:
`--check` isn't working properly for some reason. I'm testing alternatives.

Reply:
Hmm, not a fan of this the git syntax. It's very ambiguous that it searches recursive.